### ISSUE TITLE
Does not work with Babel 7

### ISSUE THREAD

**Issue by adam-digirati** [mopedjs/babel-plugin-import-globals#677]
*Error:* This API has been removed. If you're looking for this functionality in Babel 7, you should import the '@babel/helper-module-imports' module and use the functions exposed from that module, such as 'addNamed' or 'addDefault'.

This error is thrown by @babel/core/lib/transformation/file/file.js:173:11 ...

{
  "presets": [
    [
      "babel-preset-gatsby",
      {
        "targets": {
          "browsers": [">0.25%", "not dead"]
        }
      }
    ]
  ],
  "plugins": [
    [
      "import-globals",
      {
        "Link": "gatsby"
      }
    ]
  ]
}

**Comment by otaviosoares** [comment]
There is a PR for this #676 but it's been there for a while without any feedback.

**Comment by adam-digirati** [comment]
Yeah, it turns out there are a couple of forks. Apparently, the owner ( @ForbesLindesay ) abandoned the repo. The last 'human' commit was 1 Jun 2017. In case if he is not willing to maintain the codebase it would be nice if he could give rights to one of the guys who had a PR.

But maybe, there's a different reason, to not update. Maybe it is a bad practice to do imports like this. Or there's an official alternative built into babel, which I still can't find. But either of these cases, he didn't communicate it properly.
